### ISSUE TITLE
Fix SEO/Crawler issues with the Admin Login link

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#3057]
## Issue

The Admin Login link (saml/login) is showing up in search results. We can't control the destination, but we can do some things to prevent Google doesn't show this in search results.

Update:
This has recently been identified as a path that is abused by SiteImprove and other crawlers.

## Proposed Solution

- ~~Update robots.txt to cover saml/login and any paths off of it (e.g. https://sitenow.uiowa.edu/saml/login?destination=/)~~
  - Use https://www.google.com/webmasters/tools/robots-testing-tool?hl=en&utm_campaign=+6062598&siteUrl=https://sitenow.uiowa.edu/ to test changes
- Add `rel=”nofollow”` attribute to the link itself.
- The intranet split login might as well have this too.
- Consider a rule in .htaccess for certain user_agents

**Comment by joewhitsitt** [comment]
Not sure if this is still an issue. Tested the URL and got "URL is not available to Google
This page cannot be indexed. Pages that aren't indexed can't be served on Google."

**Comment by joewhitsitt** [comment]
I don't think this is an issue anymore since https://github.com/uiowa/uiowa/pull/709 and search results had a chance to catch up.

**Comment by joewhitsitt** [comment]
Re-opening because we now need attention on this from a spam perspective.

**Comment by briand44** [comment]
Grooming conversation... We previously updated robots.txt to disallow the `saml/login`. Siteimprove is suppose to stop hitting these paths as well. We still want to add the "nofollow" to the admin login in the footer and on SiteNow sites behind authentication. Discussed holding off on any `.htaccess` rules at this point.
